_data/site.yaml

```
@@ -3,6 +3,12 @@ email: RestoreCouncil@restorethegulf.gov
 description: >-
   The Gulf Coast Ecosystem Restoration Council (RESTORE Council) is a small federal agency established in the wake of the Deepwater Horizon disaster to plan for and carry out restoration across the Gulf Coast.
 
+alert:
+  enabled: true
+  heading: "Public Comment Period:"
+  text: Draft 2026 FPL Oct. 15 - 11:59 pm MT November 14, 2025
+  url: /
```

I think we can combine these so that we can have flexibility to use content that may or may not include links. We should be able to pass HTML in here
```suggestion
  text: Draft 2026 FPL Oct. 15 - 11:59 pm MT November 14, 2025. <a href="/">Submit a public comment</a>
```